### PR TITLE
Initial version of console script and entry point

### PR DESCRIPTION
**Pull request by jlstevens**
This PR aims to get a working version of the console script quickly and will be superseded by the more complete approach in #30.

Currently the console script supports the following:

```
pyviz --install-examples examples-dir
```

TODO:

- [x] Make the download sample data option work.
- [ ] Make sure it is robust and tested.

### CODE REVIEW

File: pyviz/cmd.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+-------------
+
+$ pyviz --install-examples pyviz-examples
+
+This will place the examples in the 'pyviz-examples' directory.
+
+$ pyviz --download-sample-data data
+
+This will download the sample data to the 'data' directory.
+"""
```
Comment (jbednar) [inline pyviz/cmd.py:21]: Presumably `description` and `epilog` should be the first things in the file, as they function as docstrings for the file.

**Comment by jbednar** [comment]
Looks good so far.

File: pyviz/cmd.py (hunk 1 above)
Comment (jlstevens) [reply pyviz/cmd.py:21]: Agreed. This PR is still WIP.

**Comment by jlstevens** [comment]
@jbednar Everything now seems to be working as intended. Ready for testing/review/merge.

It would be good to test on windows to make sure nothing crops up there...

**Comment by jbednar** [comment]
Tests are failing because Travis doesn't have a copy of the large datasets, but that's unrelated to this change so I will go ahead and merge.  Travis needs to be set up to ignore those two files (or ideally to substitute a tiny version for CI instead).

**Pull request merged**